Pathfinder.elm

```diff
@@ -47,8 +50,12 @@ obstacleTiles obstacles =
   |> List.concat
 
 
-find : (Int, Int) -> List (Int, Int) -> (Int, Int) -> (Int, Int) -> List (Int, Int)
-find = astar
+createMap : List (Int, Int) -> (Int, Int) -> Map
+createMap = Astar.createGrid
+
+
+find : (Int, Int) -> (Int, Int) -> Map -> List (Int, Int)
+find = Astar.findPath
```

Comment:
@aryszka sure we can publish it separately, but still we don't need `Pathfinder.elm` because it only proxies `Astar.elm`, we can import `Astar.elm` directly